ppr-ui/src/App.vue

```diff
@@ -401,6 +405,18 @@ export default class App extends Mixins(AuthMixin) {
       } else {
         throw new Error('Invalid auth roles')
       }
+
+      const subscribedProducts = await fetchAccountProducts(this.getAccountId)
+      if (subscribedProducts) {
+        this.setUserProductSubscriptions(subscribedProducts)
+
+        const activeProductCodes = subscribedProducts
+          .filter(product => product.subscriptionStatus === ProductStatus.ACTIVE)
+          .map(product => product.code)
+        this.setUserProductSubscriptionsCodes(activeProductCodes)
+      } else {
+        throw new Error('Unable to get Products for the User')
```

Comment:
Will this always throw for staff?

Reply:
Will need to update it to take into consideration that Staff might not have an account id.

Reply:
Yessir good call, i don't believe they do. They have `BCOL_STAFF_PARTY_CODE` afaik, SBC staff might have an account, not 100% sure. 
But yes, you are correct, we will need to consider staff in these conditions.

Reply:
Added a check for accountId. If a user doesn't have an id (ie staff) then don't fetch the products.

Reply:
Excellent 👍